tools/ci/tc/tasks/test.yml

```diff
@@ -480,3 +480,10 @@ tasks:
       schedule-if:
         run-job:
           - wptrunner_infrastructure
+
+  - sink-task:
```

Comment:
I tried using `built-in/success` for this in https://github.com/web-platform-tests/wpt-tc-checks/pull/6, but it would require changing `decision.py` as that assumes each task has a command (see https://community-tc.services.mozilla.com/tasks/fhFkf2ngR0mYilsTWVj1RA/runs/0/logs/https%3A%2F%2Fcommunity-tc.services.mozilla.com%2Fapi%2Fqueue%2Fv1%2Ftask%2FfhFkf2ngR0mYilsTWVj1RA%2Fruns%2F0%2Fartifacts%2Fpublic%2Flogs%2Flive.log)

Reply:
Seems like it requires a hack in decision.py anyway; arguably we could just not write this in the yaml and make it a fully special case.

Reply:
This could work; I've been playing with it in the latest version of https://github.com/web-platform-tests/wpt-tc-checks/pull/6

However, I've been having trouble getting it to work, still getting permission errors: https://community-tc.services.mozilla.com/tasks/PKBeaTZTRxeO8eEDNUBCqg/runs/0/logs/https%3A%2F%2Fcommunity-tc.services.mozilla.com%2Fapi%2Fqueue%2Fv1%2Ftask%2FPKBeaTZTRxeO8eEDNUBCqg%2Fruns%2F0%2Fartifacts%2Fpublic%2Flogs%2Flive.log

Maybe the schedulerId is wrong?

Reply:
Ah, dustin on TC Matrix told me that the decision task itself would need `queue:create-task:highest:built-in/success` permission.

Reply:
Ok, I've got something that works now, albeit the code could do with cleanup. There's some interesting things:

- The task fails the validation test, I believe (I expect the unittest to fail), even though it can be successfully created on Taskcluster.
- There's some reuse of data that was in tests.yml (deadline, schedulerId), haven't decided yet how to most cleanly handle this.

But worth a first look, so PTAL :)